env.go

```diff
@@ -114,6 +114,11 @@ func get(field reflect.StructField) (string, error) {
 	defaultValue := field.Tag.Get("envDefault")
 	val = getOr(key, defaultValue)
 
+	expandVar := field.Tag.Get("ExpandVar")
```

Comment:
I think it should be `envExpand` to be in the same pattern of the other options.

Adding it to the readme as well would be nice =)

Reply:
Good idea. I've updated and simplified the overall change once again.